resources/js/Pages/MissingOrg.vue

```diff
@@ -0,0 +1,103 @@
+<script lang='ts'>
+  import Layout from '../Shared/Layout.vue';
+  import Vue from 'vue';
+  import InputField from '../Shared/InputField.vue';
+  import SwitchBtn from '../Shared/SwitchBtn.vue';
+
+  interface MissingOrgType {
+    orgVal: string,
+    useOrg: boolean
+  }
+
+  export default Vue.extend({
+    name: 'Home',
+    components: {
+      Layout,
+      InputField,
+      SwitchBtn
+    },
+    data(): MissingOrgType {
+      return {
+        orgVal: '',
+        useOrg: false
+      };
+    },
+    methods: {
+      handleSwitch(e: Event): void {
+        this.useOrg = e;
+      },
+      getPayload(): string {
+        const payload: any = {
+          value: this.orgVal,
+          is_org: this.useOrg
+        };
+
+        return JSON.stringify(payload);
+      },
+      orgVals(e: any): void {
+        this.orgVal = e;
+      },
+      submitPayload(): void {
+        try {
+          const payload: string = this.getPayload();
+          this.$inertia.post('/org/create', payload, {
+            replace: false,
+            preserveState: true,
+            preserveScroll: false,
+            _token: document.querySelector('meta[name="csrf-token"]')?.getAttribute('content')
+          });
+        } catch (e) {
+          alert(e.message);
```

Comment:
I imagine we'll do something similar to the Alert component you made for orman but this works for now

Reply:
Totally, instead of using the alert component I wanted to look into Flash messages.